ports/libusbp/vcpkg.json

```
@@ -0,0 +1,18 @@
+{
+  "name": "libusbp",
+  "version": "1.3.1",
+  "description": "The Pololu USB Library (also known as libusbp) is a cross-platform C library for accessing USB devices.",
+  "homepage": "https://github.com/pololu/libusbp",
+  "license": "MIT",
```

```suggestion
  "license": "MIT",
```
Sorry, I don't see any place upstream that applies the `MIT` license. Could you please explain? ：）

---

Sure! The contents of https://github.com/pololu/libusbp/blob/master/LICENSE.txt match the contents of the MIT license text (https://opensource.org/license/mit)